packages/twilio-run/src/serverless-api/utils.ts

```diff
@@ -35,6 +37,18 @@ export async function getFunctionServiceSid(
   if (username) {
     debug('Attempting to read serviceSid from a deployinfo file');
     const deployInfoCache = getDeployInfoCache(cwd);
+    if (
+      deployInfoCache &&
+      deployInfoCache[`${username}:${region}`] &&
+      deployInfoCache[`${username}:${region}`].serviceSid
```

Comment:
can we use optional chaining here?  `if (deployInfoCache[`${username}:${region}`]?.serviceSid) ...`

Reply:
Ideally, this project should support node12 and above (from readme)
I would like to know Dom's opinion as node12 is deprecated already.

Reply:
I'm okay with moving to Node 14 now. I think that's outdated in the README since twilio-run itself already only supports Node 14 since a few versions ago